LOADERS/LOADER.py

```diff
@@ -0,0 +1,42 @@
+from flojoy import flojoy, JobResultBuilder
+from flojoy.utils import PlotlyJSONEncoder
+import os
+import requests
+import json
+from pathlib import Path
+import yaml
+from datetime import datetime
+
+FRONTIER_HOST= os.environ.get('FRONTIER_HOST')
+FORNTIER_PORT = os.environ.get('FRONTIER_PORT')
+API_URI= f"https://{FRONTIER_HOST}:{FORNTIER_PORT}/measurements"
+
+
+@flojoy
+def LOADER(v, params):
+    api_key = get_api_key()
+    measurement_uuid = params['measurement_uuid']
+    
+    if api_key != '' and measurement_uuid != '':
+        requests.post(f'{API_URI}/{measurement_uuid}', 
+                      json={'api_key':api_key, 
+                            'measurements': json.dumps({'data':v}, cls=PlotlyJSONEncoder),
+                            'time': datetime.now().__str__()})
+    return JobResultBuilder().from_inputs(v).build()
+
+
+def get_api_key():
+    home = str(Path.home())
+    api_key = ''
+    path = os.path.join(home, '.flojoy/credentials')
+    if os.path.exists(path):
+        stream = open(path, 'r')
+        yaml_dict = yaml.load(stream, Loader=yaml.FullLoader)
+        if isinstance(yaml_dict, str) == True:
+            split_by_line = yaml_dict.split('\n')
+            for line in split_by_line:
+                if 'FRONTIER_API_KEY' in line:
+                    api_key = line.split(':')[1]
+        else:
```

Comment:
how do we let the user know if no API key is found?

Reply:
might want to add a check in `flojoy.sh` also and alert the user that their API key has not been set

each of these error and warning messages should have a link to the frontier page where the user sets their api key

Reply:
I have added check functionality in `flojoy` script file for the Frontier API key and also updated `LOADER` node to raise an error if any of these two keys are not found.